2018-03-12 09:43:13 Error: [MissingConnectionException] Database connection "Mysql" is missing, or could not be created.
Exception Attributes: array (
  'class' => 'Mysql',
  'message' => 'SQLSTATE[HY000] [2002] Connection refused',
  'enabled' => true,
)
Request URL: /pwBoPayment/login
Stack Trace:
#0 /var/www/html/pwpayment/lib/Cake/Model/Datasource/DboSource.php(271): Mysql->connect()
#1 /var/www/html/pwpayment/lib/Cake/Model/ConnectionManager.php(105): DboSource->__construct(Array)
#2 /var/www/html/pwpayment/lib/Cake/Model/Model.php(3641): ConnectionManager::getDataSource('default')
#3 /var/www/html/pwpayment/lib/Cake/Model/Model.php(1162): Model->setDataSource('default')
#4 /var/www/html/pwpayment/lib/Cake/Model/Model.php(3668): Model->setSource('user_master')
#5 /var/www/html/pwpayment/lib/Cake/Model/Model.php(3039): Model->getDataSource()
#6 /var/www/html/pwpayment/lib/Cake/Model/Model.php(3011): Model->_readDataSource('first', Array)
#7 /var/www/html/pwpayment/lib/Cake/Controller/Component/Auth/BaseAuthenticate.php(130): Model->find('first', Array)
#8 /var/www/html/pwpayment/lib/Cake/Controller/Component/Auth/FormAuthenticate.php(79): BaseAuthenticate->_findUser('su', 'e51e52d2a830c13...')
#9 /var/www/html/pwpayment/lib/Cake/Controller/Component/AuthComponent.php(771): FormAuthenticate->authenticate(Object(CakeRequest), Object(CakeResponse))
#10 /var/www/html/pwpayment/lib/Cake/Controller/Component/AuthComponent.php(611): AuthComponent->identify(Object(CakeRequest), Object(CakeResponse))
#11 /var/www/html/pwpayment/src/Controller/LoginController.php(99): AuthComponent->login()
#12 [internal function]: LoginController->login()
#13 /var/www/html/pwpayment/lib/Cake/Controller/Controller.php(508): ReflectionMethod->invokeArgs(Object(PwBoPaymentController), Array)
#14 /var/www/html/pwpayment/lib/Cake/Routing/Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#15 /var/www/html/pwpayment/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(PwBoPaymentController), Object(CakeRequest))
#16 /var/www/html/pwpayment/webroot/index.php(118): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#17 {main}
2018-03-12 09:43:13 Error: [MissingConnectionException] Database connection "Mysql" is missing, or could not be created.
Exception Attributes: array (
  'class' => 'Mysql',
  'message' => 'SQLSTATE[HY000] [2002] Connection refused',
  'enabled' => true,
)
Request URL: /pwBoPayment/login
Stack Trace:
#0 /var/www/html/pwpayment/lib/Cake/Model/Datasource/DboSource.php(271): Mysql->connect()
#1 /var/www/html/pwpayment/lib/Cake/Model/ConnectionManager.php(105): DboSource->__construct(Array)
#2 /var/www/html/pwpayment/lib/Cake/Model/Model.php(3641): ConnectionManager::getDataSource('default')
#3 /var/www/html/pwpayment/lib/Cake/Model/Model.php(1162): Model->setDataSource('default')
#4 /var/www/html/pwpayment/lib/Cake/Model/Model.php(3668): Model->setSource('user_master')
#5 /var/www/html/pwpayment/lib/Cake/Model/Model.php(3039): Model->getDataSource()
#6 /var/www/html/pwpayment/lib/Cake/Model/Model.php(3011): Model->_readDataSource('first', Array)
#7 /var/www/html/pwpayment/lib/Cake/Controller/Component/Auth/BaseAuthenticate.php(130): Model->find('first', Array)
#8 /var/www/html/pwpayment/lib/Cake/Controller/Component/Auth/FormAuthenticate.php(79): BaseAuthenticate->_findUser('su', 'e51e52d2a830c13...')
#9 /var/www/html/pwpayment/lib/Cake/Controller/Component/AuthComponent.php(771): FormAuthenticate->authenticate(Object(CakeRequest), Object(CakeResponse))
#10 /var/www/html/pwpayment/lib/Cake/Controller/Component/AuthComponent.php(611): AuthComponent->identify(Object(CakeRequest), Object(CakeResponse))
#11 /var/www/html/pwpayment/src/Controller/LoginController.php(99): AuthComponent->login()
#12 [internal function]: LoginController->login()
#13 /var/www/html/pwpayment/lib/Cake/Controller/Controller.php(508): ReflectionMethod->invokeArgs(Object(PwBoPaymentController), Array)
#14 /var/www/html/pwpayment/lib/Cake/Routing/Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#15 /var/www/html/pwpayment/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(PwBoPaymentController), Object(CakeRequest))
#16 /var/www/html/pwpayment/webroot/index.php(118): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#17 {main}
2018-03-16 12:21:14 Error: Fatal Error (1): Call to private method Key::GetContent() from context 'LoginController' in [/var/www/html/pwpayment/src/Controller/LoginController.php, line 234]
2018-03-16 12:21:14 Error: Fatal Error (1): Call to private method Key::GetContent() from context 'LoginController' in [/var/www/html/pwpayment/src/Controller/LoginController.php, line 234]
2018-03-16 12:21:14 Error: [FatalErrorException] Call to private method Key::GetContent() from context 'LoginController'
Request URL: /pwBoPayment/generateReceipt
Stack Trace:
#0 /var/www/html/pwpayment/lib/Cake/Error/ErrorHandler.php(212): ErrorHandler::handleFatalError(1, 'Call to private...', '/var/www/html/p...', 234)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to private...', '/var/www/html/p...', 234, Array)
#2 /var/www/html/pwpayment/lib/Cake/Core/App.php(971): call_user_func('ErrorHandler::h...', 1, 'Call to private...', '/var/www/html/p...', 234, Array)
#3 /var/www/html/pwpayment/lib/Cake/Core/App.php(944): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2018-03-16 12:21:14 Error: [FatalErrorException] Call to private method Key::GetContent() from context 'LoginController'
Request URL: /pwBoPayment/generateReceipt
Stack Trace:
#0 /var/www/html/pwpayment/lib/Cake/Error/ErrorHandler.php(212): ErrorHandler::handleFatalError(1, 'Call to private...', '/var/www/html/p...', 234)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to private...', '/var/www/html/p...', 234, Array)
#2 /var/www/html/pwpayment/lib/Cake/Core/App.php(971): call_user_func('ErrorHandler::h...', 1, 'Call to private...', '/var/www/html/p...', 234, Array)
#3 /var/www/html/pwpayment/lib/Cake/Core/App.php(944): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
